code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -4618,7 +4618,7 @@
 			to_chat(M, SPAN_GOOD(pick("You feel great!", "You feel full of energy!", "You feel alert and focused!")))
 
 /decl/reagent/drink/zorasoda/overdose(var/mob/living/carbon/M, var/alien, var/datum/reagents/holder)
-	if(alien == !(IS_DIONA || IS_VAURCA)
+	if(!(alien == IS_DIONA || !(alien == IS_VAURCA)))
```

Comment:
As discussed on Discord [here](https://discordapp.com/channels/157516682288562176/302201161044328450/946432388492300308) this check does not do what you want it to.

Suggest changing to : `!(alien in list(IS_DIONA, IS_VAURCA))` here and elsewhere.

Reply:
fixed.